cmd/xl-sets.go

```diff
@@ -738,6 +738,12 @@ func (s *xlSets) ListObjects(bucket, prefix, marker, delimiter string, maxKeys i
 				}
 				return result, toObjectErr(err, bucket, prefix)
 			}
+			if objInfo.IsEncrypted() {
```

Comment:
I guess it is possible to calculate decrypted size inside  getObjectInfo(), wouldn't it be better ?